server/src/main/java/org/apache/iotdb/db/qp/executor/QueryProcessExecutor.java

```diff
@@ -234,6 +234,32 @@ public boolean insert(InsertPlan insertPlan)
     }
   }
 
+  @Override
+  public List<Integer> insertBatch(BatchInsertPlan batchInsertPlan) throws ProcessorException {
+    try {
+      String[] measurementList = batchInsertPlan.getMeasurements();
+      String deviceId = batchInsertPlan.getDeviceId();
+      MNode node = mManager.getNodeByDeviceIdFromCache(deviceId);
+
+      for (String s : measurementList) {
+        if (!node.hasChild(s)) {
+          throw new ProcessorException(
+              String.format("Current deviceId[%s] does not contains measurement:%s",
```

Comment:
`does not contains`  -> `does not contain`

Reply:
fixed